WalletWasabi.Tests/UnitTests/BlockchainAnalysis/PubKeyReuseAnonScoreTests.cs

```diff
@@ -35,6 +35,45 @@ public void AddressReusePunishment()
 			Assert.True(tx.WalletOutputs.First().HdPubKey.AnonymitySet < 30);
 		}
 
+		[Fact]
+		public void AddressReusePunishmentProcessTwice()
+		{
+			// If there's reuse in input and output side, then output side didn't gain, nor lose anonymity.
+			var analyser = ServiceFactory.CreateBlockchainAnalyzer();
+			var km = ServiceFactory.CreateKeyManager();
+			var reuse = BitcoinFactory.CreateHdPubKey(km);
+			var tx = BitcoinFactory.CreateSmartTransaction(
+				9,
```

Comment:
I know you follow the same conventions as in the other tests but anyway, my suggestion here is:

```suggestion
				othersInputCount: 9,
```

Feel free to keep your version, if you like it better.

(MSVS allows you to add that name automatically via CTRL+., if you by any chance use that IDE.)

Reply:
Done